snaxc/dialects/dart.py

```diff
@@ -1,6 +1,7 @@
 from collections.abc import Iterable, Mapping, Sequence
-from typing import Generic, TypeVar
+from typing import Generic
 
+from typing_extensions import TypeVar
```

Comment:
This took me a while to figure out.
The default one in typing is only available from python 3.13